src/Omnipay/Realex/Message/AuthRequest.php

```diff
@@ -10,7 +10,6 @@
  */
 class AuthRequest extends RemoteAbstractRequest
 {
-    protected $endpoint = 'https://epage.payandshop.com/epage-remote.cgi';
```

Comment:
Would it not make more sense to leave the endpoint property in place, and allow the setter and getter to set it/access it? Also by removing it will break the existing Global Iris driver, which extends these requests (see https://github.com/coatesap/omnipay-global-iris/blob/master/src/Omnipay/GlobalIris/Message/AuthRequest.php for example). If there is no real reason to remove it, I'd rather not have to remove/upgrade/lock down the Global Iris driver...

Reply:
I sort of agree with this implementation. This makes it possible to inject the parameters at the configuration stage (as per the merchant ID etc). Obviously the omnipay interfaces need to be adhered to so this results in not being able to explicitly pass the endpoint other than in the params.